mcs/class/System/System.Diagnostics/PerformanceCounterCategory.cs

```diff
@@ -283,12 +300,12 @@ public static bool InstanceExists (string instanceName, string categoryName, str
 			CheckCategory (categoryName);
 			if (machineName == null)
 				throw new ArgumentNullException ("machineName");
-			int val = InstanceExistsInternal (instanceName, categoryName, machineName);
-			if (val == 0)
-				return false;
-			if (val == 1)
-				return true;
-			throw new InvalidOperationException ();
+
+			//?FIXME: machine appears to be wrong
```

Comment:
This whole comment doesn't make a lot of sense and we should just return false if `machine != "."`.

Reply:
This is what the previous code was doing. I'm just preserving the behavior. I don't know if it is safe to change.

Reply:
Preserving preeexisting behavior. Don't know if it safe to fix, don't know if there is test coverage, don't know if there is compat burden. I realize it is probably safe, machine != "." malfunctions so much otherwise, hard to imagine anyone sneaks it through here. But just preserving existing behavior with hopefully slightly better easier to maintain code (more C#, less native)

Reply:
👍